app/screens/settings/notification_settings_mobile/notification_settings_mobile.android.js

```diff
@@ -21,8 +22,17 @@ import SectionItem from 'app/screens/settings/section_item';
 import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
 
 import NotificationSettingsMobileBase from './notification_settings_mobile_base';
+import {getNotificationProps} from '../../../utils/notify_props';
```

Comment:
Could you change this to be imported from `app/utils/notify_props`? We usually only use relative imports when the other file is in the same folder as this one.

Reply:
Done.